themes/openy_themes/openy_rose/Gemfile.lock

```diff
@@ -0,0 +1,71 @@
+GEM
```

Comment:
no need for lock file if we added it to .gitignore
Or let's remove it from .gitignore if we should keep the file in repo